### PR TITLE
Improve/Update README

### PR DESCRIPTION
**Pull request by DragonEggBedrockBreaking**
- Fixes link
- Clarifies meaning of sentences
- Updates based on new information

When merging, squash-merge this. Supercedes #20, #18, #9

### CODE REVIEW

**Comment by DragonEggBedrockBreaking** [comment]
cc: @hxr404 (partly contribured) @Brottweiler (gave me some suggestions)

**Comment by DragonEggBedrockBreaking** [comment]
This also PRs to the 1.17 branch unlike the last few PRs which were created pre-1.17.

**Comment by DragonEggBedrockBreaking** [comment]
Fixed a bunch of formatting issues now.

File: README.md [hunk 1]
```diff
@@ -1,23 +1,33 @@
 ## Enhanced Block Entities
 
-EBE is a **100% client side** mod for Minecraft on the **[Fabric](https://fabricmc.net/use/)** mod loader which aims to increase the performance of block entity rendering, as well as offer customizability via resource packs. <br/><br/>
-
-**How does it work?** EBE Makes chests (and soon to be more block entities) use baked block models rather than laggy entity models. <br/><br/>
+EBE is a **100% client side** Minecraft mod for the **[Fabric](https://fabricmc.net/use/)** mod loader which aims to increase the performance of block entity rendering, as well as offer customizability via resource packs. <br/><br/>
+**How does it work?** EBE Makes some block entities use baked block models rather than laggy entity models. <br/><br/>
+**Is it just an optimisation mod?** No, it also allows you to customise more block entities with resource packs and makes their lighting smoother. This mod also allows for toggling christmas chests among other nice improvements.<br/><br/>
```
Comment (FoundationGames) [inline README.md]: European spelling "optimisation" -> American "optimization"? (Personal preference)

File: README.md [hunk 2]
```diff
@@ -1,23 +1,33 @@
...
 **What about animations?** The best part about EBE is that you still get to keep animations, while gaining the performance boost of baked models! Most animated block entity models will only render when absolutely necessary. <br/><br/>
-**Can I use it with Sodium?** Yes you can, but it requires some setup. I have a [guide](https://github.com/FoundationGames/EnhancedBlockEntities/discussions/7) that you can follow to get this mod performing as optimally as possible with Sodium. <br/><br/>
-
+**Can I use it with Sodium?** Yes you can, but it requires you to install the [Indium mod](https://modrinth.com/mod/indium) as well. <br/><br/>
```
Comment (FoundationGames) [inline README.md]: Remove "mod" from the hyperlink

File: README.md [hunk 3]
```diff
@@ -1,23 +1,33 @@
...
+- Signs
+- Bells
+- Beds
+<br/><br/>
 
 Requires [Fabric API](https://modrinth.com/mod/fabric-api) <br/><br/>
 
 ## FAQ and Help
+
+**Q: How do I download the mod?**<br/>
```
Comment (FoundationGames) [inline README.md]: I think a section for download is better than making it a FAQ question

File: README.md [hunk 4]
```diff
@@ -1,23 +1,33 @@
...
+**A:** For stable releases, you can check out the [Modrinth Page](https://modrinth.com/mod/OVuFYfre). If you want the newest bleeding edge build, you can use GitHub Actions (or alternatively, you can build yourself).<br/><br/>
+
 **Q: I need help with the mod/need to report a bug!** <br/>
 **A:** If you're having trouble setting up the mod or using it alongside other mods, I'd recommend you join our [Discord Server](https://discord.gg/7Aw3y4RtY9) and ask for help there. *If the issue is a BUG* please report it on our issue tracker ("Issues" tab at the top of the page)<br/><br/>
 
 **Q: My chests are invisible!** <br/>
-**A:** You're using Sodium, which doesn't support certain Fabric Rendering features by default. If you want to use Sodium with EBE, you should follow [this guide](https://github.com/FoundationGames/EnhancedBlockEntities/discussions/7). <br/><br/>
+**A:** You're using Sodium, which doesn't support certain Fabric Rendering features by default. If you want to use Sodium with EBE, you should Install the indium mod. <br/><br/>
```
Comment (FoundationGames) [inline README.md]: Capitalize Indium and make into a hyperlink?

File: README.md [hunk 5]
```diff
@@ -1,23 +1,33 @@
...
 
 **Q: How is this different from [FastChest](https://www.curseforge.com/minecraft/mc-mods/fastchest)?** <br/>
-**A:** FastChest does not preserve chest animations, and only optimizes chests (in a similar way to EBE). EBE aims to optimize many more block entities (in future releases). <br/><br/>
+**A:** FastChest does not preserve chest animations, and only optimizes chests (in a similar way to EBE). EBE also optimizes other block entities, e.g. bells and signs. <br/><br/>
```
Comment (FoundationGames) [inline README.md]: e.g. -> such as

**Review by FoundationGames — CHANGES_REQUESTED** [review]
Thanks for all the PRs. I have a few requests for this one though.

File: README.md (hunk 1 above)
Comment (DragonEggBedrockBreaking) [reply README.md]: Spelling fixed, just a habit, sorry

Comment (DragonEggBedrockBreaking) [reply README.md]: Spelling fixed, just a habit, sorry

File: README.md (hunk 2 above)
Comment (DragonEggBedrockBreaking) [reply README.md]: Fixed this one as well

File: README.md (hunk 4 above)
Comment (DragonEggBedrockBreaking) [reply README.md]: Fixed

File: README.md (hunk 5 above)
Comment (DragonEggBedrockBreaking) [reply README.md]: I instead moved the entire list of other block entities from the main section to here, thus removing the need to use e.g.

File: README.md (hunk 3 above)
Comment (DragonEggBedrockBreaking) [reply README.md]: Done

**Comment by DragonEggBedrockBreaking** [comment]
All of the changes have been completed.

**Review by hxr404 — APPROVED** [review]
yep I think this could be merged now.
btw. thanks for the credit @DragonEggBedrockBreaking

**Pull request merged**